src/components/textfield/__tests__/__snapshots__/TextField.test.js.snap

```diff
@@ -2,10 +2,17 @@
 
 exports[`components/textfield/TextField should render the textfield 1`] = `
 <div
-  class="mb-4"
+  class="mb-4 text-secondary-light focus-within:text-primary"
```

Comment:
@crysfel: This `focus-within:text-primary` class doesn't work at all. Tailwind doesn't end up generating the CSS class for it. I've followed the instructions on the Tailwind site and added in the `focus-within` variant in `tailwind.js`, but I'm not having any luck. Do you know how to fix it?

Reply:
Just to have a little bit of context, have have you tried until now? Have you compiled it? @trent-boyd

Reply:
I didn't officially _compile_ it, but in Storybook's dev mode, the `<style>` block that Tailwind generates doesn't have `focus-within` listed anywhere in it. Other pseudo-classes work fine. It's a fairly new feature in Tailwind (2-3 months) and this is the first time we're using it.

Reply:
You need to enable it, for some reason that's not documented O.o Just add `focus-within` to the `textColors` array to `tailwind.js` file

```
textColors: ['responsive', 'hover', 'focus', 'focus-within'],
```

Reply:
@crysfel: Yep, I've already done that. No dice.

Reply:
That did the trick for me in a different project where I needed to use `focus-within`. Maybe the version we are using doesn't support it? 🤷‍♂️

Reply:
@crysfel: I figured it out. As a Tailwind n00b, I didn't know I had to have a process running to generate new styles. `yarn run wach:css` was the step I missed. 😅